website/content/v1.11/advanced/developing-talos.md

```diff
@@ -6,8 +6,7 @@ aliases:
 ---
 
 This guide outlines steps and tricks to develop Talos operating systems and related components.
-The guide assumes Linux operating system on the development host.
-Some steps might work under Mac OS X, but using Linux is highly advised.
+The guide assumes MacOS or a Linux operating system on the development host.
```

Comment:
```suggestion
The guide assumes macOS or a Linux operating system on the development host.
```

Reply:
The next rename is likely to be soon since macOS is a rather old name from 2016, so it could be rational to move that into a variable /s

macOS probably covers all the versions that work for Talos development though